### PR TITLE
Update Changelog

### PR DESCRIPTION
**Pull request by pulpbot**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -10,6 +10,21 @@
 
 [//]: # (towncrier release notes start)
 
+## 0.4.0 (2024-04-30)
+
+
+
+No significant changes.
```
Comment (mdellweg) [inline CHANGES.md:17]: Do you want to mention the compatibility bump?

Comment (lubosmj) [reply CHANGES.md:17]: Not a bad idea.

Comment (lubosmj) [reply CHANGES.md]: Does this suffice?

**Comment by lubosmj** [comment]
Thanks, pulpbot! 😠

**Comment by mdellweg** [comment]
Definitely nicer than nothing! :+1:

**Review by mdellweg — APPROVED** [review]

**Pull request merged**